### PR TITLE
feat(payments): INT-3931 Add SEPA form for Checkout.com

### PR DESCRIPTION
**Pull request by MLoyo**
[INT-3931](https://jira.bigcommerce.com/browse/INT-3931)
## What?
- Add SEPA form for Checkout.com payment method, injecting as a component in `CheckoutcomCustomPaymentMethod.tsx` and its validation schema
- Create a file to store future custom forms for Checkout.com
- Change multiple filenames in order to indicate more modularity for the multiple future forms for Checkout.com

## Why?
As one of the requirements  it's support SEPA, the form is necessary to request the necessary data to create the payment request

## Testing / Proof

https://user-images.githubusercontent.com/69487179/109849127-9ff9b900-7c16-11eb-897d-aa17e16caa07.mov

## Related PRs

- bigcommerce/checkout-sdk-js#1079

@bigcommerce/checkout

### CODE REVIEW

**Review by FranciscoOnt — APPROVED** [review]
LGTM

File: src/app/locale/translations/en.json [hunk 1]
```diff
@@ -185,6 +185,7 @@
             "chasepay_edit_card": "Edit card",
             "chasepay_logout": "Log out Chase Pay",
             "chasepay_fail_load": "Chase Pay failed to load",
+            "checkoutcom_sepa_mandate_disclaimer":"By accepting this mandate form, you authorize {storeUrl} and Checkout.com, our payment service provider, to send instructions to your bank to debit your account, and your bank to debit your account in accordance with those instructions. You are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited.",
```
Comment (capsula4) [inline src/app/locale/translations/en.json:188]: might it be better to use storeName rather than storeUrl ?

**Review by capsula4 — APPROVED** [review]

File: src/app/locale/translations/en.json (hunk 1 above)
Comment (MLoyo) [reply src/app/locale/translations/en.json:188]: In the ticket is specified that string template, and also `payment.stripe_sepa_mandate_disclaimer` has a similar structure, let me ask I'll fix it if it's necessary

Comment (MLoyo) [reply src/app/locale/translations/en.json]: I just updated the PR following your suggestion
Thanks C:

**Comment by PascalZajac** [comment]
@MLoyo Circle seems to have missed running on this PR, I can't find how to trigger it manually. Can you try rebasing?

**Pull request merged**